src/Stryker.Core/Stryker.Core/Mutators/Mutator.cs

```diff
@@ -33,7 +33,7 @@ public enum Mutator
         String,
         [Description("Bitwise operators")]
         Bitwise,
-        [Description("Array initializer")]
+        [Description("Initializer")]
```

Comment:
This creates a breaking change. To avoid that we might support multiple descriptions for a mutator. We can then deprecate the old description and remove it in the next breaking change.

Reply:
Revert this back to Array initializer

Reply:
Reverted 👍